code/modules/client/preference_setup/loadout/lists/uniforms.dm

```diff
@@ -186,3 +186,7 @@
 /datum/gear/uniform/battledress_german
 	display_name = "battle dress uniform, oberth"
 	path = /obj/item/clothing/under/germansuit
+
+/datum/gear/uniform/syndicate
+	display_name = "Syndicate turtleneck"
+	path = /obj/item/clothing/under/syndicate
```

Comment:
No newline.

Reply:
what?

Reply:
Check [this](https://lmgtfy.app/?q=why+file+should+end+with+newline).

Reply:
> Check [this](https://lmgtfy.app/?q=why+file+should+end+with+newline).

ah i thought you meant i shouldnt newline. allright added one